### PR TITLE
Fix button group

### PR DESCRIPTION
**Pull request by oTkPoBeHuE**

### CODE REVIEW

File: packages/urban-bot/src/utils/formatButtonElement.ts [hunk 1]
```diff
@@ -10,24 +10,34 @@ export type FormattedButton = OtherProps & {
     onClick?: (...args: unknown[]) => unknown;
 };
 
+type ButtonsElement = React.ReactElement<ButtonProps>;
```
Comment (ledamint) [inline packages/urban-bot/src/utils/formatButtonElement.ts]: ButtonElement

File: packages/urban-bot/src/utils/formatButtonElement.ts [hunk 2]
```diff
@@ -10,24 +10,34 @@ export type FormattedButton = OtherProps & {
...
+
+function isButtonMatrix(
```
Comment (ledamint) [inline packages/urban-bot/src/utils/formatButtonElement.ts]: isButtonsMatrix

File: packages/urban-bot/src/components/ButtonGroup.tsx [hunk 3]
```diff
@@ -14,7 +14,7 @@ export type ButtonGroupProps = UrbanMessageCommonData & {
     isResizedKeyboard?: boolean;
     isNewMessageEveryRender?: boolean;
     maxColumns?: number;
-    children: React.ReactNode;
+    children: React.ReactElement<ButtonProps> | React.ReactElement<ButtonProps>[] | React.ReactElement<ButtonProps>[][];
```
Comment (ledamint) [inline packages/urban-bot/src/components/ButtonGroup.tsx]: ButtonGroup can accept not only Buttons

File: packages/urban-bot/src/utils/formatButtonElement.ts (hunk 1 above)
Comment (oTkPoBeHuE) [reply packages/urban-bot/src/utils/formatButtonElement.ts]: Fixed

File: packages/urban-bot/src/utils/formatButtonElement.ts (hunk 2 above)
Comment (oTkPoBeHuE) [reply packages/urban-bot/src/utils/formatButtonElement.ts]: Fixed

**Pull request merged**